ddl/ddl_api.go

```diff
@@ -5810,6 +5810,9 @@ func buildPlacementSpecs(bundle *placement.Bundle, specs []*ast.PlacementSpec) (
 		case ast.PlacementRoleFollower:
 			role = placement.Follower
 		case ast.PlacementRoleLeader:
+			if spec.Replicas == 0 {
+				spec.Replicas = 1
+			}
```

Comment:
What if the user sets it to 0 explicitly?
e.g. `leader=role replicas=0`

Reply:
This case has been treated in the `parser`.

Reply:
why not just throw "Invalid placement option REPLICAS, it is should be identified"? @djshow832

Reply:
The number of leaders can be only 1. It's unnecessary to declare it.

Reply:
ok